bin/install

```diff
@@ -391,18 +406,20 @@ installIngressGatewayCertificate() {
 installSimulator() {
   check ${FUNCNAME[0]}
 
-  if [[ ! -d ${configs[simulator]} ]]; then
-    mkdir -p ${configs[workdir]}
-    (cd ${configs[workdir]};
-      git clone ${configs[simulator-repository]} ${configs[simulator]})
-  fi
+  if [ "${configs[simulator-repository]}" ]; then
+    if [[ ! -d ${configs[simulator]} ]]; then
+      mkdir -p ${configs[workdir]}
+      (cd ${configs[workdir]};
+        git clone ${configs[simulator-repository]} ${configs[simulator]})
+    fi
 
-  (export PATH=/usr/local/opt/python/libexec/bin:${PATH};
-    cd ${configs[simulator]}; \
-    git pull; \
-    pip install -r requirements.txt; \
-    docker build -t trackgen .;
-    helm install ./helm --name trackgen)
+    (export PATH=/usr/local/opt/python/libexec/bin:${PATH};
```

Comment:
This path is not going to exist for a lot of people

Reply:
likely for non-brew installs ... will take it into consideration

Reply:
inferring install path from $(brew --prefix python)/libexec/bin

Reply:
It looks like the install path in linux is `/home/linuxbrew/.linuxbrew/bin`

Reply:
ok thx. i will work through this with the linux notebook in hand as well.

Reply:
ok $(brew --prefix python)/bin/python exists on osx and linux